tree/ntuple/v7/inc/ROOT/RNTupleProcessor.hxx

```diff
@@ -270,6 +303,21 @@ public:
    static std::unique_ptr<RNTupleProcessor>
    CreateChain(const std::vector<RNTupleOpenSpec> &ntuples, std::unique_ptr<RNTupleModel> model = nullptr);
 
+   /////////////////////////////////////////////////////////////////////////////
+   /// \brief Create a new RNTuple processor chain for vertical combinations of RNTuples.
```

Comment:
The docstring here and in the `Create` overloads for the single RNTupleProcessor could be made a bit more symmetric, i.e. if for the single processor we have

> Create an `RNTupleProcessor` for a single RNTuple.

for the chain processor could be

> Create an `RNTupleProcessor` that vertically combines multiple RNTuples in a chain.

Or something similar